ThumbnailItem/ThumbnailItem.js

```diff
@@ -40,12 +91,18 @@ const ThumbnailItemBase = kind({
 		publicClassNames: true
 	},
 
-	render: ({children, css, src, ...rest}) => {
+	computed: {
+		className: ({type, styler}) => styler.append({
+			roundThumbnail: type === 'round'
+		})
+	},
+
```

Comment:
I might not give you enough information to fix your branch.
I expected that the `render` would be

```
	render: ({children, css, src, ...rest}) => {
		return (
			<Item {...rest}>
				<slotBefore className={css.slotBefore}>
					<img
						className={css.thumbnail}
						src={src}
					/>
				</slotBefore>
				<div className={css.content}>
					{children}
				</div>
			</Item>
		);
	}
```

Then the `rest` including the `label` will be passed to the `Item`. If you have any concern to do it, please let me know.

Reply:
I modified the `render` as you suggested.